scripts/o3de/ExportScripts/export_standalone_monolithic_project_centric.py

```diff
@@ -27,9 +27,9 @@
 # Example invocation on Linux
 '''
 $ cd <project-path>
-$ <engine-path>/scripts/o3de.sh export-project --export-scripts ExportScripts/export_standalone_monolithic.py \
+$ <engine-path>/scripts/o3de.sh export-project --export-scripts <engine-path>/scripts/o3de/ExportScripts/export_standalone_monolithic_project_centric.py \
  --project-path . --log-level INFO --output-path <install-path> --build-non-mono-tools --config release --archive-output xz \
- --seedlist ./Assets/seedlist1.seed --seedlist ./Assets/seedlist2.seed <etc..> 
+ --seedlist ./Assets/seedlist1.seed --seedlist ./Assets/seedlist2.seed <etc..> -code -assets
```

Comment:
What are the -code -assets arguments?

Reply:
They are used to package assets and code. Without at least one, the script (appears to?) do nothing. I added both, since anyone wanting to package their game probably wants to run both (at least for the first run).

Reply:
We could just make those arguments true by default. @tkothadev what do you think?

Reply:
I think that makes sense. Usually the project export is performed with the intent of building the code and assets. But it'd require negating the current arguments to make sense (something like `no-code` and `no-assets`)